ops/charm.py

```diff
@@ -697,10 +697,10 @@ def from_yaml(
                 This can be a simple string, or a file-like object. (passed to `yaml.safe_load`).
             actions: YAML description of Actions for this charm (eg actions.yaml)
         """
-        meta = _yaml.safe_load(metadata)
+        meta = yaml.safe_load(metadata)
```

Comment:
Do we ever end up with a conflict about what the 'yaml' module is? I just wonder about people wondering why they don't have the rest of PyYAML available.

Reply:
Yeah, I wondered about that. But my intention here is that `from ops._private import yaml` will be a drop-in replacement for `import yaml`, *for the subset of functionality we need*. In this case that's only `safe_load` and `safe_dump`, but if we need to add more names, like `YAMLError`, we can easily do that later. You happy with that?

Reply:
I think so. It isn't meant to be used outside of this project, which is the most important part.

Reply:
Yep, that's right. And we'll get loud `AttributeError`s raised if someone does try to use something that doesn't exist in `_private/yaml.py`.